content/en/security/application_security/troubleshooting.md

```diff
@@ -369,6 +369,9 @@ Enable debug logs with the environment variable `DD_TRACE_DEBUG=1`. The ASM libr
 
 **Note:** ASM only outputs logs when it is enabled. Use the environment variable `DD_APPSEC_ENABLED=1` to enable ASM.
 
+#### Crashes in go-libddwaf code
+
+If you see segfaults happening during `github.com/DataDog/go-libddwaf` module execution, please consider enabling CGO using the environment variable `CGO_ENABLED=1` during compilation. If the issue persist, please consider opening an issue..
```

Comment:
```suggestion
If you see segfaults happening during `github.com/DataDog/go-libddwaf` module execution, please consider enabling CGO using the environment variable `CGO_ENABLED=1` during compilation. If the issue persist, please consider opening an issue.
```

Reply:
Thanks for the review 🙏  in the end we decided to remove this new section